can_tools/scrapers/official/OH/oh_vaccine_demographics.py

```diff
@@ -1,22 +1,123 @@
-    location_name_col = ""
     timezone = "US/Eastern"
+    url = r"https://public.tableau.com/views/VaccineAdministrationMetricsDashboard/PublicCountyDash?:embed=y&:showVizHome=no&:host_url=https%3A%2F%2Fpublic.tableau.com%2F&:embed_code_version=3&:device=desktop&:tabs=no&:toolbar=no&:showAppBanner=false&iframeSizedToWindow=true&:loadOrderID=0"
+    demographic_col_name = "Race"
+    demographic = "race"
+    rename_key = {
+        "black or african american": "black",
+        "american indian alaska native": "ai_an",
+        "native hawaiian pacific islander": "pacific_islander",
+        "multiracial": "multiple",
+    }
 
     # map wide form column names into CMUs
-    cmus = {}
+    variables = {
+        "initiated": variables.INITIATING_VACCINATIONS_ALL,
+        "complete": variables.FULLY_VACCINATED_ALL,
+    }
+
+    def _extract_data(self, shot_type: str) -> pd.DataFrame:
+        ts = TableauScraper()
+        ts.loads(self.url)
+
+        # set type of metric and demographic (initiated is fetched by default)
+        book = ts.getWorkbook()
+        # get all possible paramaters
+        params = pd.DataFrame(book.getParameters())
+        shot_vals = params.loc[params["column"] == "View By", "values"].iloc[0]
+        demo_vals = params.loc[params["column"] == "Key Metrics", "values"].iloc[0]
+        assert self.demographic_col_name in demo_vals
+        book.setParameter("Key Metrics", self.demographic_col_name)
+
+        if shot_type == "complete":
+            print(f"set parameter to: {shot_vals[1]}")
+            book.setParameter("View By", shot_vals[1])
+
+        parts = []
+        ws = ts.getWorksheet("New Map")
+        counties = ws.getSelectableValues("county")
+        for county in counties:
+            print("working on county: ", county)
+            wb = ws.select("county", county)
+            df = wb.getWorksheet("New Demographics").data.assign(location_name=county)
+            df = df.rename(columns={"SUM(Chosen Dose Status Metric)-alias": shot_type})
+            assert county in wb.getWorksheet("New County or Statewide").data.iloc[0, 0]
+            parts.append(df)
+            ws = ts.getWorksheet("New Map")
+        return pd.concat(parts)
+
+    def fetch(self) -> tuple:
+        init = self._extract_data("initiated")
+        complete = self._extract_data("complete")
+        return init, complete
+
+    def normalize(self, data: TableauScraper) -> pd.DataFrame:
+        df = pd.merge(
+            data[0],
+            data[1],
+            how="left",
+            on=["Selected Demographic-alias", "location_name"],
+        )
+        df = df[
+            ["location_name", "Selected Demographic-alias", "initiated", "complete"]
+        ]
+        df["Selected Demographic-alias"] = df["Selected Demographic-alias"].str.lower()
+        if self.rename_key:
+            df = df.replace(self.rename_key)
+
+        out = self._reshape_variables(df, self.variables)
+        out = (
```

Comment:
looks like this might be repeating some of the work done in `._reshape_variables`?

In particular this code does `out = self._reshape_variables(df, self.variables)` and then overrides `out` when we do `out =(df.melt)` -- i.e. the first `out` isn't used.

Reply:
I see what you did -- I will just delete the `_reshape_variables` call as it is not needed

Reply:
whoops--thanks for the catch. I was trying to see if I could use `._reshape_variables` with demographic columns then must have forgotten to delete it afterwards